runners_manager/metrics/Runner.py

```diff
@@ -0,0 +1,11 @@
+"""File defining metrics object for Runners."""
+
+from prometheus_client import Enum
+
+
+PROMETHEUS_RUNNER_STATUS = Enum(
+    'openstack_actions_runner_status', 'Metrics displaying the status of a runner',
+    states=['creating', 'deleting', 'respawning', 'online', 'running', 'offline'],
+    labelnames=['name', 'flavor', 'image'],
+)
+
```

Comment:
knowing which repo is being executed for each runner would be great, but I think it can be added on later on,

Reply:
Regarding the class, I don't know if we should yet, I feel like people tend not to. But I may be wrong.

Have a look at the documentation:
* https://prometheus.io/docs/instrumenting/writing_clientlibs/
* https://github.com/prometheus/client_python 

I feel like we should use those objects as vars inside their dedicated class. Let me know what you think

Reply:
I'm gonna wait for your thoughts, but I think I'm gonna move this var to be inside the runner class.

Reply:
Event prometheus, in there Python client, give class for more complex tracking: 
- `from prometheus_client.multiprocess import MultiProcessCollector`
- `from prometheus_client import CollectorRegistry`

Because as they say "Take advantage of each language’s features." When we will start to have more complex dashboard with in a way duplicate data in different format, i am certain this will help.

Reply:
Added the metrics inside a class.